fasttrips/Passenger.py

```diff
@@ -509,7 +509,10 @@ def setup_passenger_pathsets(self, iteration, stops, trip_id_df, trips_df, modes
         `pf_iteration`                int64  iteration in which these paths were found
         `pathnum`                     int64  the path number for the path within the pathset
         `pf_cost`                   float64  the cost of the entire path
+        `pf_fare`                   float64  the fare of the entire path
```

Comment:
The difference of "cost" and "fare" is not initially obvious.
Perhaps we should use Util instead of cost?  Other thoughts?